charters/DataProcessingService/DataProcessingPipelines.md

```diff
@@ -0,0 +1,70 @@
+# [HCA Data Processing Pipelines](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+The HCA Data Processing Pipelines are cloud-based and portable analysis pipelines that transform the data in the Human Cell Atlas.
+
+## Objectives
+The data processing service aims to develop scientific analysis pipelines for all analyses required on all scientific data types in the Human Cell Atlas Data Coordination Platform (DCP). Pipelines for additional data types, analyses, and improvements to these pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community. 
+
+## Definitions
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports
+__Onboarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay 
+__Standardized Pipeline:__ An official pipeline of the HCA project 
+
+## In-scope
+* Scoping or review of scoping for new pipelines  
+* Creating all standardized pipelines for scientific data 
+* Reviewing, on-boarding, and potentially creating community pipelines  
+* Reviewing community tools for use in data processing pipelines 
+* Creating tools that support the creation of community pipelines - may include software, documentation or support
+* Updating pipelines ran in the HCA DCP for optimization of cost, time, or scientific benchmarks  
+* Establishing standard benchmarking used to evaluate data processing pipelines  
```

Comment:
This is a general tool/infrastructure piece (unity), right?

Reply:
This is more around the benchmarking test suites that would be put into Unity, there is a separate charter for Unity itself.